ts/Stock/Navigator/Navigator.ts

```diff
@@ -438,45 +442,46 @@ class Navigator {
                 cursor: inverted ? 'ns-resize' : 'ew-resize'
             },
             // Create the main navigator group
-            navigatorGroup: SVGElement = navigator.navigatorGroup = renderer
-                .g('navigator')
-                .attr({
-                    zIndex: 8,
-                    visibility: 'hidden'
-                })
-                .add();
+            navigatorGroup: SVGElement = isDirty ? navigator.navigatorGroup :
+                navigator.navigatorGroup = renderer
+                    .g('navigator')
+                    .attr({
+                        zIndex: 8,
+                        visibility: 'hidden'
+                    })
+                    .add();
```

Comment:
Maybe instead of basing on `isDirty` flag, we wanna use either nullish coalescing, or simple or, to determine if the property is already defined?

Other idea would be splitting the creation to different method, but it might be a lot of work 🤔